CITATION.cff

```diff
@@ -0,0 +1,88 @@
+cff-version: 1.0.0
+message: "If you use this software, please cite it as below."
+type: software
+title: "Software Lifecycle Improvement & Modernization (SLIM)"
+abstract: "A shared community resource for discussing, iterating and referencing a common set of best practices in software development standards for NASA multi-mission space and ground software."
+authors:
```

Comment:
The authors list is always evolving and there could be some folks contributing more in words than code. Can we remove the authors list entirely and just keep the citation to the SLIM project itself?